CompetitionMars/CompetitionMars/Pages/EducationPage.cs

```diff
@@ -0,0 +1,257 @@
+﻿using CompetitionMars.Utilities;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using CompetitionMars.DataModel;
+using System.Reflection.Emit;
+using System.Collections.ObjectModel;
+
+namespace CompetitionMars.Pages
+{
+    public class EducationPage : CommonDriver
+    {
+        private static IWebElement addNewButton => driver.FindElement(By.XPath("//*[text()='Country']//parent::tr//child::th[6]//div"));
+        private static IWebElement universityNameTextbox => driver.FindElement(By.Name("instituteName"));
+        private static IWebElement countryDropdown => driver.FindElement(By.Name("country"));
+        private static IWebElement titleDropdown => driver.FindElement(By.Name("title"));
+        private static IWebElement degreeTextbox => driver.FindElement(By.Name("degree"));
+        private static IWebElement yearOfGraduationDropdown => driver.FindElement(By.Name("yearOfGraduation"));
+        private static IWebElement addButton => driver.FindElement(By.XPath("//input[@value='Add']"));
+        private static IWebElement cancelBButton => driver.FindElement(By.XPath("//input[@value='Cancel']"));
+        private static IWebElement educationTab => driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[1]/a[3]"));
+        
+        private static IWebElement addedTitle = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/table/tbody[last()]/tr/td[3][last()]"));
+        //private static IWebElement addedDegree = driver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[4]/div/div[2]/div/table/tbody/tr/td[4][last()]"));
```

Comment:
remove unwanted comments every where, its is bad practice to check in commented out code